.env.example

```diff
@@ -1,7 +1,7 @@
 APP_NAME="The Accessibility Exchange"
 APP_ENV=development
 APP_KEY=
-APP_DEBUG=false
+APP_DEBUG=true
```

Comment:
Is this an intentional change?

Reply:
Oh, this isn't intentional, I accidentally commit this file.